### DISCUSSION TITLE
Google map not loading when I insert the tarteaucitron script

### DISCUSSION THREAD

**Discussion by gilles66** [AmauriC/tarteaucitron.js#1252] [Q&A] [answered]
I am currently creating a website with WordPress, using a plugin that generates a google map with a shortcode.

This google map isn't loaded as soon as I insert the tarteaucitron js script. 
And no error message in the console log !  

A page with script tarteaucitron not inserted
https://yyourlocaleye.perpi.bz/produit/sejour-en-auvergne/

A page with script tarteaucitron inserted
https://yyourlocaleye.perpi.bz/produit/sejour-en-auvergne/?tarteaucitron=1

The map should be just above "Produits similaires".

Anyone an idea ?

**Comment by AmauriC** [comment, marked as answer]
Google maps need consent so the widget is blocked until the consent.

To fix the load of the Maps when tarteaucitron allow maps, add this Javascript snippet:

```
document.addEventListener('googlemaps_loaded', function() {
   setTimeout(function() {
      tarteaucitron.addScript("https://yyourlocaleye.perpi.bz/wp-content/plugins/yith-woocommerce-booking-premium/modules/google-maps/assets/js/booking-map.min.js");
   }, 800);
});
```

**Comment by gilles66** [comment]
This worked, thanks a lot !
